README.md

```diff
@@ -28,10 +29,15 @@ The primary goal of Coverband is giving deep insight into your production runtim
 * Development mode, offers deep insight of code usage details (number of LOC execution during single request, etc) during development.
 * Mountable web interface to easily share reports
 
-
 # Installation
 
-Coverband should be near zero config for Rails apps, if you have an issue with that, please [file an issue](https://github.com/danmayer/coverband/issues).
+## Redis
```

Comment:
Thought we could use mention of redis a bit earlier since it is required.

Reply:
makes sense